app/index.js

```diff
@@ -10,6 +10,8 @@ const querystring = require('querystring')
 const { config } = require(path.join(__dirname, 'package.json'))
 const swarm = require('./swarm')
 
+const USE_TESTNET = process.env.USE_TESTNET || true
```

Comment:
Shouldn't we default to `false` to match the server behavior of running on mainnet by default?

Reply:
I just left this is as true for now whilst using testnet, should of put a note there to change default when ready for Mainnet